composer.json

```
@@ -10,11 +10,21 @@
         }
     ],
     "require": {
-        "psr/log": "~1.0"
+        "php": "8.0",
```

```suggestion
        "php": ">=8.0",
```
I had to change this otherwise got this error:
```
  Problem 1
    - Root composer.json requires php 8.0 but your php version (8.0.29) does not satisfy that requirement.
```